spec/expectations/odk/forms/repeat_group_form.xml

```diff
@@ -80,6 +80,7 @@
       <label>Repeat Group 1</label>
       <repeat nodeset="/data/*itemcode1*">
         <group appearance="field-list">
+          <label>Group Item Label</label>
```

Comment:
I think this should be a `ref` style label like the others, and the actual text should go in the `itext` block. I know we aren't doing $ notation in this issue but we still need to include the different translations. Right now if you add french it won't show up in the XML at all.

Reply:
this will just use xpath as a ref here. 1) fix existing expectations to use itext for group name labels 2) update the translation group node to render itext 3) update group body to use the itext.